aries-site/src/pages/components/avatar.mdx

```diff
@@ -0,0 +1,111 @@
+import { BestPracticeGroup, Example } from '../../layouts';
+import {
+  AvatarAccessibilityExample,
+  AvatarCardExampleDo,
+  AvatarCardExampleDont,
+  AvatarDataTableExampleDo,
+  AvatarDataTableExampleDont,
+  AvatarPageHeaderExample,
+  AvatarSizePreviewExample,
+} from '../../examples';
+
+## Anatomy
+
+Avatars can be displayed as an image, initials, or a logo. They come in three different sizes:
+
+- `small`: 24x24px
+- `medium`: 48x48px
+- `large`: 96x96px
+
+The container of an Avatar can be circular or a rounded square based on the use case.
+
+- Use a circle Avatar when representing a person either through an image or their initials.
+- Use a rounded square Avatar when representing a company’s or entity’s logo. This can be achieved by passing `round='medium'`
```

Comment:
```suggestion
- Use a rounded square Avatar when representing a company’s or entity’s logo. This can be achieved by applying `round='medium'` to the Avatar.
```